prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,32 @@
+const number1 = 22;
+const number2 = 19;
+const product = number1 * number2;
+console.log('value of product:', product);
+console.log('typeof product:', typeof product);
+const charge = 19.79;
+const payment = 12.57;
+const amountRemaining = payment - charge;
+console.log('value of amountRemaining:', amountRemaining);
+console.log('typeof amountRemaining:', typeof amountRemaining);
+const tests = 55;
+const assignments = 25;
+const final = 20;
+const grade = (tests + assignments + final) / 3;
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+const firstName = 'Ryan';
+const lastName = 'Garcia';
+const fullName = firstName + ' ' + lastName;
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+const pH = 3;
+const isAcidic = 'ph' <= 7;
+console.log('value of isAcidic', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+const numberOfSoldiers = 1000;
+const isSparta = numberOfSoldiers === 300;
+console.log('value of isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+let nickname = fullName + ' ' + 'is the GOAT';
```

Comment:
This should be broken up into 2 lines. You should originally assign `fullName` to the variable `nickName`. And then on a line underneath, use the `+=` operator to add the `'is the GOAT'` to `nickname`